docs/sql-ref-datatypes.md

```diff
@@ -157,6 +159,8 @@ You can access them by doing
 |**DoubleType**|Double|DoubleType|
 |**DecimalType**|java.math.BigDecimal|DecimalType|
 |**StringType**|String|StringType|
+|**CharType(length)**|String|CharType(length)|
+|**VarcharType(length)**|String|VarcharType(length)|
```

Comment:
I'm a bit hesitant to document the,. They are not real data types yet. Once read from the table, they become StringType with special annotations.

Reply:
Please take a look at L40